src/views/Users/ViewUser/index.jsx

```diff
@@ -164,6 +164,10 @@ function ViewUser({ isNewUser, ...props }) {
                 // Array destructuring is not very readable here.
                 // eslint-disable-next-line prefer-destructuring
                 permission.sc = sc[0];
+
+                if (permission.sc.options === null) {
```

Comment:
Could this be `undefined`? I wonder if we should just do `if (!permission.sc.options)` which would catch both `undefined` and `null`.

Reply:
Yeah, that's a good idea.